src/redux/reducer.js

```diff
@@ -354,29 +356,13 @@ export const reducer = createReducer(initialState, {
 });
 
 function synchWorkingNodeAndSelectedNode(state) {
```

Comment:
This function doesn't sync workingNode anymore, rename it or I don't know... Maybe it's because workingNode will be removed soon ?

Reply:
ill keep managing synchronization of both kind of nodes (selected, working) until we decide witch solution is better to do the refactor (example: using one type of node in global scope of the app)